backend/src/main/resources/application.yaml

```diff
@@ -55,6 +55,7 @@ simple-report-initialization:
       model: LumiraDx SARS-CoV-2 Ag Test*
       loinc-code: "95209-3"
 simple-report:
```

Comment:
Claiming our own top-level property namespace is a good practice that I should have followed with my previous local property code. Thanks for doing it this way, and I shall move the initializer stuff in here when I get a chance!